app/i18n/es.json

```diff
@@ -626,6 +626,7 @@
     "confirmSubtext": "Este nombre de usuario es permanente y no se podrá cambiar más adelante.",
     "confirmTitle": "¿Desea establecer %{input} como su nombre de usuario?",
     "letterAndNumber": "Solo se aceptan letras en minúscula, números y guiones bajos (_)",
+    "forbiddenStart": "No se puede empezar con bc1, 1, o 3 ",
```

Comment:
maybe is better to use "No debe empezar con bc1, 1 o 3 "

Reply:
Also list "forbiddenStart" before "letterAndNumber" to maintain alphabetical order.

Reply:
Done!